docs/rst/developer_manual/installation/sources/windows.rst

```diff
@@ -426,36 +372,30 @@ Local installation
 Global installation
 -------------------
 
-To install |ddsrecorder| system-wide instead of locally, remove all the flags that
-appear in the configuration steps of :code:`Fast-CDR`, :code:`Fast-DDS`, :code:`Dev-Utils`, :code:`DDS-Router`, and
-:code:`DDS-Recorder`
+To install |eddsrecord| system-wide instead of locally, remove all the flags that appear in the configuration steps of :code:`Fast-CDR`, :code:`Fast-DDS`, :code:`Dev-Utils`, :code:`DDS-Pipe`, and :code:`DDS-Recorder`
```

Comment:
```suggestion
To install |eddsrecord| system-wide instead of locally, remove all the flags that appear in the configuration steps of :code:`Fast-CDR`, :code:`Fast-DDS`, :code:`Dev-Utils`, :code:`DDS-Pipe`, and :code:`DDS-Record`
```